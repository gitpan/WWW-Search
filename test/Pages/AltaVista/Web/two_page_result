http://www.altavista.com/cgi-bin/query?what=web&text=yes&fmt=d&pg=q&q=%2BLSAM+%2BISI+%2BIB
/cgi-bin/query?pg=q&text=yes&q=%2BLSAM+%2BISI+%2BIB&stq=10&c9k
