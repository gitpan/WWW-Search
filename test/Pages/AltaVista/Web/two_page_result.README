This set of test-result pages was created on Wed Sep 29 16:33:26 1999
