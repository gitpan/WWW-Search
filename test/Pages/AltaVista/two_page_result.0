<html><head>
<title>AltaVista: Simple Query +LSAM +ISI +IB</title>
</head><body>
<font size=+1 face=arial, helvetica>
<a href="/cgi-bin/query?text">AltaVista Search Service</a>
</font>
<br>
<FORM name=mfrm method=GET action="/cgi-bin/query">
<INPUT TYPE=hidden NAME=pg VALUE=q>
<INPUT TYPE=hidden NAME=text VALUE=yes>
<font size=-1><B>Ask AltaVista<sup><small>TM</small></sup> a question.</B>&nbsp;&nbsp;Or enter a few words in
<SELECT NAME=kl><OPTION VALUE=XX SELECTED>any language 
<OPTION VALUE=zh>Chinese
<OPTION VALUE=cs>Czech
<OPTION VALUE=da>Danish
<OPTION VALUE=nl>Dutch
<OPTION VALUE=en>English
<OPTION VALUE=et>Estonian
<OPTION VALUE=fi>Finnish
<OPTION VALUE=fr>French
<OPTION VALUE=de>German
<OPTION VALUE=el>Greek
<OPTION VALUE=he>Hebrew
<OPTION VALUE=hu>Hungarian
<OPTION VALUE=is>Icelandic
<OPTION VALUE=it>Italian
<OPTION VALUE=ja>Japanese
<OPTION VALUE=ko>Korean
<OPTION VALUE=lv>Latvian
<OPTION VALUE=lt>Lithuanian
<OPTION VALUE=no>Norwegian
<OPTION VALUE=pl>Polish
<OPTION VALUE=pt>Portuguese
<OPTION VALUE=ro>Romanian
<OPTION VALUE=ru>Russian
<OPTION VALUE=es>Spanish
<OPTION VALUE=sv>Swedish
</SELECT></font><br><INPUT NAME=q size=50 maxlength=800 VALUE="+LSAM +ISI +IB">
<br><INPUT TYPE=radio NAME=act VALUE=search CHECKED>search <INPUT TYPE=radio NAME=act VALUE=refine>refine <INPUT TYPE=submit VALUE=Submit><br>
 <P><font size=-1>Search - <a href="/cgi-bin/query?pg=aq&what=web&text=yes">Advanced</a> - <a href="/cgi-bin/query?pg=q&what=news&text=yes">Usenet</a></font><BR></FORM>
<font size=-1><b>AltaVista found 18 Web pages for you. </font></b><P><font size=-1><b>Result Pages: </b><b>1</b> &nbsp;<a href="/cgi-bin/query?pg=q&text=yes&q=%2bLSAM+%2bISI+%2bIB&stq=10&c9k">2</a>
&nbsp; <a href="/cgi-bin/query?pg=q&text=yes&q=%2bLSAM+%2bISI+%2bIB&stq=10&c9k">[<b>Next &gt;&gt;</b>]</a> <br>
</font><font size=-1><dl><dt><b>1. </b><a href="http://www.isi.edu/ib/example.html"><b>IB Research Plan</b></a><dd>
IB Research Plan. For the experiments in this research, we will make use of traffic generators that emulate WWW browser request streams, to evaluate the...<br><b>URL:</b> <font color=gray>www.isi.edu/ib/example.html<br>
Last modified 13-Jan-97 - page size 6K - in English</font> [&nbsp;<a href="http://jump.altavista.com/trans.go?urltext=http%3a%2f%2fwww%2eisi%2eedu%2fib%2fexample%2ehtml&language=en">Translate</a>&nbsp;]</dl>
<dl><dt><b>2. </b><a href="http://www.isi.edu/ib/"><b>About IB</b></a><dd>
About Intelligent BW. In today's Internet, information services such as WWW, Gopher, WAIS, etc. are oriented towards a reactive model: the Internet and...<br><b>URL:</b> <font color=gray>www.isi.edu/ib/<br>
Last modified 6-Oct-97 - page size 3K - in English</font> [&nbsp;<a href="http://jump.altavista.com/trans.go?urltext=http%3a%2f%2fwww%2eisi%2eedu%2fib%2f&language=en">Translate</a>&nbsp;]</dl>
<dl><dt><b>3. </b><a href="http://www.isi.edu/div7/cities/ib/tuner/about.html"><b>About IB</b></a><dd>
About the Multicast Web Tuner. Proxy Benefits: Move files closer to client to improve response speed when files are requested again. Share proxies so that.<br><b>URL:</b> <font color=gray>www.isi.edu/div7/cities/ib/tuner/about.html<br>
Last modified 12-Feb-98 - page size 4K - in English</font> [&nbsp;<a href="http://jump.altavista.com/trans.go?urltext=http%3a%2f%2fwww%2eisi%2eedu%2fdiv7%2fcities%2fib%2ftuner%2fabout%2ehtml&language=en">Translate</a>&nbsp;]</dl>
<dl><dt><b>4. </b><a href="http://www.isi.edu/~lsam/ib/rmc/investigate.html"><b>IB - Reliable Multicast Transport protocol</b></a><dd>
IB - Reliable Multicast Transport protocol. AFDP. Dynamically selects from available transmission modes so as to maximize efficiency and throughput while..<br><b>URL:</b> <font color=gray>www.isi.edu/~lsam/ib/rmc/investigate.html<br>
Last modified 12-Feb-98 - page size 3K - in English</font> [&nbsp;<a href="http://jump.altavista.com/trans.go?urltext=http%3a%2f%2fwww%2eisi%2eedu%2f%7elsam%2fib%2frmc%2finvestigate%2ehtml&language=en">Translate</a>&nbsp;]</dl>
<dl><dt><b>5. </b><a href="http://www.isi.edu/~lsam/ib/rmc/links.html"><b>IB - Reliable Multicast Transport protocol</b></a><dd>
IB - Reliable Multicast Transport protocol. Here are a few good links for Multicast Transport Protocols....<br><b>URL:</b> <font color=gray>www.isi.edu/~lsam/ib/rmc/links.html<br>
Last modified 12-Feb-98 - page size 1K - in English</font> [&nbsp;<a href="http://jump.altavista.com/trans.go?urltext=http%3a%2f%2fwww%2eisi%2eedu%2f%7elsam%2fib%2frmc%2flinks%2ehtml&language=en">Translate</a>&nbsp;]</dl>
<dl><dt><b>6. </b><a href="http://www.isi.edu/ib/discussion.html"><b>IB Discussion</b></a><dd>
IB Discussion. Response time and bandwidth are key resources in this project. Current Internet user interfaces, such as Web browsers, Gopher and WAIS...<br><b>URL:</b> <font color=gray>www.isi.edu/ib/discussion.html<br>
Last modified 13-Jan-97 - page size 6K - in English</font> [&nbsp;<a href="http://jump.altavista.com/trans.go?urltext=http%3a%2f%2fwww%2eisi%2eedu%2fib%2fdiscussion%2ehtml&language=en">Translate</a>&nbsp;]</dl>
<dl><dt><b>7. </b><a href="http://www.isi.edu/div7/lsam/ib/projects.html"><b>IB Research Plan</b></a><dd>
IB Research Projects. Lowlat. Anticipatory document caching reduces request latency. Multicast Web Tuner. Multicasting popular documents on channels...<br><b>URL:</b> <font color=gray>www.isi.edu/div7/lsam/ib/projects.html<br>
Last modified 12-Feb-98 - page size 2K - in English</font> [&nbsp;<a href="http://jump.altavista.com/trans.go?urltext=http%3a%2f%2fwww%2eisi%2eedu%2fdiv7%2flsam%2fib%2fprojects%2ehtml&language=en">Translate</a>&nbsp;]</dl>
<dl><dt><b>8. </b><a href="http://www.isi.edu/div7/lsam/ib/about.html"><b>About IB</b></a><dd>
About Intelligent BW. Motivations. Ethernet or faster bandwidth is needed to interactively view the average web page. Bandwidth requirements increase as...<br><b>URL:</b> <font color=gray>www.isi.edu/div7/lsam/ib/about.html<br>
Last modified 15-Jan-98 - page size 3K - in English</font> [&nbsp;<a href="http://jump.altavista.com/trans.go?urltext=http%3a%2f%2fwww%2eisi%2eedu%2fdiv7%2flsam%2fib%2fabout%2ehtml&language=en">Translate</a>&nbsp;]</dl>
<dl><dt><b>9. </b><a href="http://www.isi.edu/div7/lsam/ib/rmc/req.html"><b>IB - Reliable Multicast Transport protocol</b></a><dd>
IB - Reliable Multicast Transport protocol. Reliability Mechanism. Retransmission is event driven (off-line). Retransmission requests uses selactive NACKS.<br><b>URL:</b> <font color=gray>www.isi.edu/div7/lsam/ib/rmc/req.html<br>
Last modified 12-Feb-98 - page size 2K - in English</font> [&nbsp;<a href="http://jump.altavista.com/trans.go?urltext=http%3a%2f%2fwww%2eisi%2eedu%2fdiv7%2flsam%2fib%2frmc%2freq%2ehtml&language=en">Translate</a>&nbsp;]</dl>
<dl><dt><b>10. </b><a href="http://www.isi.edu/div7/ib/plan.html"><b>IB Research Plan</b></a><dd>
IB Research Plan. For the experiments in this research, we will make use of traffic generators that emulate WWW browser request streams, to evaluate the...<br><b>URL:</b> <font color=gray>www.isi.edu/div7/ib/plan.html<br>
Last modified 13-Jan-97 - page size 4K - in English</font> [&nbsp;<a href="http://jump.altavista.com/trans.go?urltext=http%3a%2f%2fwww%2eisi%2eedu%2fdiv7%2fib%2fplan%2ehtml&language=en">Translate</a>&nbsp;]</dl>
</font><font size=-1><b>Result Pages: </b><b>1</b> &nbsp;<a href="/cgi-bin/query?pg=q&text=yes&q=%2bLSAM+%2bISI+%2bIB&stq=10&c9k">2</a>
&nbsp; <a href="/cgi-bin/query?pg=q&text=yes&q=%2bLSAM+%2bISI+%2bIB&stq=10&c9k">[<b>Next &gt;&gt;</b>]</a> <br>
</font><font size=-2>word count: LSAM: 874; ISI: 130362; IB: 169529<BR></font><P>
<br>
<font size=-2 face=arial, helvetica>
<a href="/cgi-bin/query?text">AltaVista Home</a> | 
<a href="/av/content/help.htm">Help</a> | 
<a href="/av/content/questions.htm">Feedback</a>
<br> 
Copyright | 
<a href="/av/content/disclaimer.htm">Disclaimer</a> | 
<a href="/av/content/privacy.htm">Privacy</a>
<br>
<a href="/av/content/about.htm">About AltaVista</a> | 
<a href="/cgi-bin/query?pg=tmpl&v=pref.html">Set your Preferences</a> | 
<a href="/av/content/addurl.htm">Add a Page</a> | 
<a href="/cgi-bin/query">Graphic Version</a>
</font>
</BODY></HTML>
