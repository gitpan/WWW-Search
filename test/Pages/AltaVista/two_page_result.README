This set of test-result pages was created on Fri Oct  8 10:56:59 1999
