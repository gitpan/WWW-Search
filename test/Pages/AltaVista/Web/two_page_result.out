http://www.isi.edu/ib/
http://www.isi.edu/lsam/ib/about.html
http://www.isi.edu/div7/ib/plan.html
http://www.isi.edu/lsam/ib/index.html
http://www.isi.edu/div7/ib/example.html
http://www.isi.edu/lsam/ib/people.html
http://www.isi.edu/div7/ib/discussion.html
http://theory.uwinnipeg.ca/CPAN/data/WWW-Search/test/Lycos/README.html
http://xent.ics.uci.edu/FoRK-archive/jan98/0429.html
http://www.isi.edu/lsam/ib/related-work.html
http://www.isi.edu/~lsam/about/
http://www.darpa.mil/ito/Summaries97/C370_0.html
